flow/project.py

```diff
@@ -349,12 +372,12 @@ def cmd(self):
 
     def set_status(self, value):
         "Store the operation's status."
-        self.job._project.document.setdefault('_status', dict())[self.id] = int(value)
+        self.jobs[0]._project.document.setdefault('_status', dict())[self.id] = int(value)
```

Comment:
For now (if we allow only one job in an aggregate) I think it's complete. If you wan't I can start implementing the behaviour we discussion (storing the ids for status check)

Reply:
I think it's fine to leave this for the current PR since it's 100% backwards compatible. We just need to track this carefully so that we change it in a future PR.

Reply:
Yes, I'll definitely track this carefully for future work.